### PR TITLE
package-multiple niggles

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

File: scripts/build-json/build-json.js [hunk 1]
```diff
@@ -18,22 +17,26 @@ function walk(directory, filepaths) {
...
 }
 
-function collectItems(directory) {
+function collectItems(directory, searchPaths) {
     let filepaths = [];
     walk(directory, filepaths);
-    filepaths = filepaths.map( filepath => filepath.slice(path.join(process.cwd(), './content/').length) )
+    filepaths = filepaths.map( 
+        filepath => filepath.slice(path.join(process.cwd(), './content/').length) 
+    ).filter(filepath => !searchPaths.length || searchPaths.some(searchPath => filepath.includes(searchPath)))
```
Comment (wbamberg) [inline scripts/build-json/build-json.js:25]: To me, this line is really hard to read.

**Comment by wbamberg** [comment]
@peterbe , this look great in general. One bit I thought was hard to read, but maybe that's just me?

File: scripts/build-json/build-json.js (hunk 1 above)
Comment (peterbe) [reply scripts/build-json/build-json.js:25]: This'll get fixed once we let Prettier tidy it up.

**Comment by peterbe** [comment]
Independent of the ugly line (which'll soon be fixed) can you merge this so we can focus on *your* PR towards `master`?

**Pull request merged**